dependencies.sh

```diff
@@ -0,0 +1,35 @@
+#!/bin/sh
+
+install() {
+    install_npm_packages
+    install_composer_packages
+    install_ruby_bundler
+}
+
+install_npm_packages() {
+    echo '🤖  [Yarn / NPM installing]'
+    if command -v yarn 2>/dev/null; then
+        yarn install
+    elif command -v npm 2>/dev/null; then
+        npm i
+    else
+        echo 'No Yarn or NPM available.'
+        exit 1
+    fi
+}
+
+install_composer_packages() {
+    if command -v composer 2>/dev/null; then
+        composer install
+    else
+        echo 'No PHP Composer available.'
+        exit 1;
+    fi
+}
+
+install_ruby_bundler() {
+    gem install --user-install bundler &&
```

Comment:
Waarom doet deze als enige geen check of `gem` en `bundler` wel uitvoerbaar zijn?